### DISCUSSION TITLE
Index Temp files

### DISCUSSION THREAD

**Discussion by mikemm1022** [ravendb/ravendb#13443] [General]
I have a load of older temp files in the index\temp directory is it safe to delete these

**Comment by haludi** [comment]
Hi,

RavenDB will delete temp files by itself when you start it so you can leave
it.
*Don't delete the whole index directory.*
But you can delete the temp files (scratch & compression with the "buffers"
prefix inside the Temp folder).
[image: image.png]


 Best regards,
Igal

**Reply by ayende** [reply]
Note that those are marked as delete on close and should be automatically removed by RavenDB on shutdown.
If RavenDB is running, you shouldn't be deleting those, they will be collected at some idle point
